AlgorithmiaTests/AlgorithmiaTests.swift

```diff
@@ -13,12 +13,9 @@ class AlgorithmiaTests: XCTestCase {
     var client:AlgorithmiaClient?
     override func setUp() {
         super.setUp()
-        
-        let ALGORITHMIA_API_KEY = "" // Your API Key
-        
+        let ALGORITHMIA_API_KEY = ProcessInfo.processInfo.environment["ALGORITHMIA_API_KEY"] ?? PLACE_API_KEY// Your API Key
```

Comment:
Could you replace `PLACE_API_KEY` (and then you probably have to rename the `let` binding too). Something like:

```swift
let apiKey = ProcessInfo.processInfo.environment["ALGORITHMIA_API_KEY"] ?? ALGORITHMIA_API_KEY
```